_sass/app.scss

```diff
@@ -863,6 +918,12 @@ main {
   }
 }
 
+// category-nav to show when javascript is disabled.
```

Comment:
If we no longer need this, then let’s delete this. If we need it again then we can dig it out of git history